adoc/chapters/programming_interface.adoc

```diff
@@ -12578,65 +12580,66 @@ T identity() const
    a@ Return the identity value of the combination operation associated with
       this [code]#reducer#.  Only available if the identity value is known
       to the implementation.
-
 |====
 
 [[table.operators.reducer]]
-.Operators of the [code]#reducer# class
+.Hidden friend operators of the [code]#reducer# class
 [width="100%",options="header",separator="@",cols="65%,35%"]
 |====
 @ Operator @ Description
 a@
 [source]
 ----
-template <typename T>
-    void operator+=(reducer<T,plus<T>,0>& accum, const T& partial)
+    reducer& operator+=(reducer& accum, const T& partial)
 ----
    a@ Equivalent to calling [code]#accum.combine(partial)#.
+      Available only when: [code]#Dimensions == 0 &&
+      (is_same_v<BinaryOperation, plus<>> || is_same_v<BinaryOperation, plus<T>>)#.
```

Comment:
Should we have everywhere `std::is_same_v`?

Reply:
Should `plus`, etc. also be `std::plus`?

Reply:
If we talk about something from the `std::`, yes.